cli/cmd/deploy.go

```diff
@@ -85,12 +89,16 @@ func buildUndeployServiceCommand(srv string) *cobra.Command {
 		Short: `Undeploys a ` + srv + ` service`,
 		Long:  `Undeploys a ` + srv + ` service, removing it from a running profile, identified by its name`,
 		Run: func(cmd *cobra.Command, args []string) {
-			serviceManager := compose.NewServiceManager()
```

Comment:
What's the reasoning for moving docker compose specific calls into the deploy namespace?

Reply:
The reason was to avoid cyclic references in packages. I explained it in 9c9768dfc506b5b769dbc7316a8cbf16534d512d commit message:

>We will need another abstraction to represent the Docker client operations,
    as it's clear what is a deployment and what is an operation in the deployment.
    Maybe a Client struct for each provider will help out in differenciate it

Reply:
Ah good catch on the cyclic references!